actovos-consulting-group/LnchBx
# LNCHBX-10: removed CRA for webpack

removed CRA scripts for webpack.

To test:
- ensure your AWS profile is set to the ittybam env.
- root bootstrap the env for development
- remove node_modules and run `npm I`
- `docker-compose up -d database`
- /server `php artisan key:generate` (since the env was updated)
- /server `php artisan serve`
- /client `npm I`
- /client `npm run dev`
